guides/leon/debain-email.md

```diff
@@ -0,0 +1,308 @@
+# Debian Email Box
+
+## Summary 
+This is the setup guide for the Debian Email box. We will be using a Postfix, dovecot, and webmail setup.
+
+### Contact
+- Twitter: @Dubliyu
+- Slack: @yourlocalgod on wcscusf.slack.com
+- Email: [Gmail](mailto:cleonromero@mgail.com)
+
+### Table
+1. [Prerequisites](#pre-reqs)
+2. [Setup Debian VM](#setup-vm)
+3. [Install Postfix](#postfix)
+4. [Install Dovecot](#dovecot)
+5. [Install Webmail](#webmail)
+6. [Testing](#testing)
+
+<a id="pre-reqs"></a>
+## Prerequisites 
+1. Have your [virtual environment](https://silexone.github.io/guides/nestor/ISPsetup.html) configured
+2. Have the [ISP](https://silexone.github.io/guides/nestor/ISPsetup.html) gateway running.

```

Comment:
When this is merged in along with Nestor's and Sharon's guides make another pull request with the new links. For now this is good, just a note for the future.